frontend/pulse3d/src/pages/uploads.js

```diff
@@ -192,7 +325,9 @@ export default function Uploads() {
           .filter(({ uploadId }) => uploadId === id)
           .sort((a, b) => new Date(b.datetime) - new Date(a.datetime));
 
-        const lastAnalyzed = uploadJobs[0] ? uploadJobs[0].datetime : formattedTime;
+        const lastAnalyzed = uploadJobs[0]
+          ? uploadJobs[0].datetime
+          : formattedTime;
```

Comment:
Seems like your prettier config is setting things back to a line length of 80.. might check to see if we can move everything back to line length of 110 because we'll keep having to change the line length back and forth in PRs.